backend/src/app.ts

```diff
@@ -2,10 +2,16 @@ import * as bodyParser from "body-parser";
 import express, { Application } from "express";
 import "dotenv/config";
 import { APILogger } from "./logger/api.logger";
+import { connect } from "./config/db.config";
+import routes from "./routes";
 import * as fs from "fs";
 import { getAllModelsInDir } from "./utils";
 import * as path from "path";
+import { Sequelize } from "sequelize-typescript";
+import UserService from "./service/api";
```

Comment:
not okay with importing services every time in the `app.ts` file.
can you maybe explain this

Reply:
Yet to get a feedback for this